node/Makefile

```diff
@@ -572,8 +572,8 @@ $(WINDOWS_UPGRADE_INSTALL_ZIP): build-windows-archive $(WINDOWS_UPGRADE_DIST_STA
 # Build the docs site and copy over the install-calico-windows.ps1 script.
 $(WINDOWS_INSTALL_SCRIPT):
 	-mkdir -p dist
-	make -C ../calico clean _site
-	cp $(CURDIR)/../calico/_site/scripts/install-calico-windows.ps1 $@
+	cp $(CURDIR)/windows-packaging/install-calico-windows.ps1 $@
+	sed -i s/VERSION/$(GIT_VERSION)/g $@
```

Comment:
Maybe to get over the hash release issues we could use `LAST_GIT_TAG` defined by `git describe --tags --abbrev=0` instead of `GIT_VERSION`? It won't work for master though so there's still something more permanent that needs to be thought out.

Reply:
we can solve these issues in a follow-up PR since we need to move the docs.